flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/source/reader/fetcher/SplitFetcherManagerTest.java

```diff
@@ -100,6 +102,50 @@ void testCloseCleansUpPreviouslyClosedFetcher() throws Exception {
         fetcherManager.close(Long.MAX_VALUE);
     }
 
+    /**
```

Comment:
This comment could be removed

Reply:
It might be better to keep this comment here. In general, the tests should be testing against the behavior. But in this particular case, the test is against the implementation. Keeping the comment helps the maintainers know that this test can actually be removed if the implementation changes.